webdriver-spec.html

```diff
@@ -2441,12 +2440,8 @@
  the <a>title</a> IDL attribute of the <a>current top-level browsing
  context</a>'s <a>active document</a>.
 
- <li><p>Let <var>body</var> be a new JSON Object.
-
- <li><p>Set the property "<code>value</code>" of <var>body</var>
-  to the value of <var>title</var>.
-
- <li><p>Return <a>success</a> with data <var>body</var>.
+ <li><p>Return <a>success</a> with data being the value
+ of <var>title</var>.
```

Comment:
Same here. Please let us just say “Return success with data _title_” throughout this patch.